backend/pages/views.py

```
@@ -32,6 +32,8 @@ def is_valid_xlsx(file):
 def home(request):
     # Prepare default case
     quarters = Quarter.objects.filter(user=request.user)
+    
+    files = ExcelFile.objects.filter(user=request.user, is_processed=False)
```

Ruff (0.8.2)

Local variable `files` is assigned to but never used

Remove assignment to unused variable `files`

(F841)